waiter/src/waiter/handler.clj

```diff
@@ -419,7 +430,9 @@
                      (:time service-suspended-state)
                      (assoc :service-suspended-state service-suspended-state)
                      (seq source-tokens-entries)
-                     (assoc :source-tokens source-tokens-entries))]
+                     (assoc :source-tokens source-tokens-entries)
+                     (seq current-for-tokens)
```

Comment:
Don't we want to add this key even if `current-for-tokens` is empty?

Reply:
I was just following the convention of all the other optional stuff in the file. Even service description isn't added if it's empty or missing.